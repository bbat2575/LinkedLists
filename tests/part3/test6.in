NEW 1
1
NEW 1
2
INSERT 1 0 {0}
VIEW-NESTED 1
TYPE 1
NEW 4
1
2.2
$
bazinga
VIEW-NESTED 2
TYPE 2
INSERT 2 4 {0}
INSERT 2 -1 {0}
INSERT 2 0 {0}
INSERT 2 -8 {0}
VIEW-NESTED 2
TYPE 2
NEW 10
1
2.2
$
bazinga
this
is
a
long
list
.
NEW 1
1
INSERT 4 0 {3}
INSERT 4 -1 {3}
INSERT 4 -3 {3}
INSERT 4 3 {3}
INSERT 4 1 {0}
INSERT 4 -2 {0}
INSERT 4 3 {0}
VIEW-NESTED 4
TYPE 4
