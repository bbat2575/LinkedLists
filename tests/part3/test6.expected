
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> List 1: 2

> Nested 1: {List 0} -> 2

> {1} -> 2

> reference -> int

> List 2: 1 -> 2.20 -> $ -> bazinga

> 1 -> 2.20 -> $ -> bazinga

> int -> float -> char -> string

> Nested 2: 1 -> 2.20 -> $ -> bazinga -> {List 0}

> Nested 2: 1 -> 2.20 -> $ -> bazinga -> {List 0} -> {List 0}

> Nested 2: {List 0} -> 1 -> 2.20 -> $ -> bazinga -> {List 0} -> {List 0}

> Nested 2: {List 0} -> {List 0} -> 1 -> 2.20 -> $ -> bazinga -> {List 0} -> {List 0}

> {1} -> {1} -> 1 -> 2.20 -> $ -> bazinga -> {1} -> {1}

> reference -> reference -> int -> float -> char -> string -> reference -> reference

> List 3: 1 -> 2.20 -> $ -> bazinga -> this -> is -> a -> long -> list -> .

> List 4: 1

> Nested 4: {List 3} -> 1

> Nested 4: {List 3} -> 1 -> {List 3}

> Nested 4: {List 3} -> {List 3} -> 1 -> {List 3}

> Nested 4: {List 3} -> {List 3} -> 1 -> {List 3} -> {List 3}

> Nested 4: {List 3} -> {List 0} -> {List 3} -> 1 -> {List 3} -> {List 3}

> Nested 4: {List 3} -> {List 0} -> {List 3} -> 1 -> {List 3} -> {List 0} -> {List 3}

> Nested 4: {List 3} -> {List 0} -> {List 3} -> {List 0} -> 1 -> {List 3} -> {List 0} -> {List 3}

> {1 -> 2.20 -> $ -> bazinga -> this -> is -> a -> long -> list -> .} -> {1} -> {1 -> 2.20 -> $ -> bazinga -> this -> is -> a -> long -> list -> .} -> {1} -> 1 -> {1 -> 2.20 -> $ -> bazinga -> this -> is -> a -> long -> list -> .} -> {1} -> {1 -> 2.20 -> $ -> bazinga -> this -> is -> a -> long -> list -> .}

> reference -> reference -> reference -> reference -> int -> reference -> reference -> reference

> 
